### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- inspired by #200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Home Assistant Yellow website
+
+This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 
+
+## Setup
+
+This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.
+
+## Development
```
Comment (frenck) [inline README.md:9]: Might it be an idea to by default point to codespaces?

Especially if one would just make a single edit quickly, you don't need anything and just start typing working directly from your browser...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The easiest way to edit the documentation is to open the environment in Github Codespaces. Codespaces work in your browser, no installation required.
+1. Further up on this page, select the green **Code** button. 
+2. On the **Codespaces** tab, select **Open in codesapce**.
+3. Make your changes and push them.
+4. You are amazing! Thank you for contributing.
+
```
Comment (strugee) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+This is the source of the Yellow documentation website at [yellow.home-assistant.io](https://yellow.home-assistant.io). 
+
+## Setup
+
+This website is built using [11ty](https://www.11ty.dev/). The website is automatically deployed using Netlify.
+
+## Development
+
+The easiest way to edit the documentation is to open the environment in Github Codespaces. Codespaces work in your browser, no installation required.
```
Comment (strugee) [inline README.md:11]:
```suggestion
The easiest way to edit the documentation is to open the environment in GitHub Codespaces. Codespaces work in your browser, no installation required.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+1. Further up on this page, select the green **Code** button. 
+2. On the **Codespaces** tab, select **Open in codesapce**.
```
Comment (strugee) [inline README.md:13]:
```suggestion
2. On the **Codespaces** tab, select **Create codespace on main**.
```

Comment (strugee) [reply README.md:13]: Presumably I'm getting this button text since I've never used Codespaces for this repo before

File: README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+3. Make your changes and push them.
+4. You are amazing! Thank you for contributing.
+
+
+If you prefer to run this locally, the recommended development environment is Visual Studio Code. The repository uses [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers). When opening a checked out version of this repository, make sure to open it in the Dev Container (Ctrl+Shift+P > **Dev Containers: Open Folder in Container...**)
+
+If you prefer other editors, Dev Containers is an open standard and can be used with other editors. See [containers.dev](ttps://containers.dev/) for more information on supported editors.
```
Comment (strugee) [inline README.md:20]:
```suggestion
If you prefer other editors, Dev Containers is an open standard and can be used with other editors. See [containers.dev](https://containers.dev/) for more information on supported editors.
```

**Review by strugee — APPROVED** [review]
Left some small corrections, LGTM overall. Thank you!

**Comment by c0ffeeca7** [comment]
Thank you for the review, @strugee :+1:

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**